main.go

```diff
@@ -250,14 +252,20 @@ func main() {
 
 	// Create a function that adds all the controllers and webhooks to the manager.
 	addToManager := func(ctx *context.ControllerManagerContext, mgr ctrlmgr.Manager) error {
+		tracker, err := setupRemoteClusterCacheTracker(ctx, mgr)
+		if err != nil {
+			setupLog.Error(err, "unable to create remote cluster tracker tracker")
```

Comment:
```suggestion
```
Let's not log here. The returned error will be eventually logged in ~ l.304 anyway

(But let's use this text for error wrapping in l.258)

Reply:
Done